### PR TITLE
Ibpsa1844 move interpolate to math

### PR DESCRIPTION
**Pull request by hcasperfu**
This PR is to merge https://github.com/ibpsa/modelica-ibpsa/pull/1845 to Buildings.

### CODE REVIEW

File: Buildings/.copiedFiles.txt [hunk 1]
```diff
@@ -2185,7 +2183,8 @@ Buildings/Resources/Images/Utilities/Math/Functions/Examples/cubicHermite.png
 Buildings/Resources/Images/Utilities/Math/Functions/SmoothExponentialXPower.png
 Buildings/Resources/Images/Utilities/Math/int.pdf
 Buildings/Resources/Images/Utilities/Math/int.png
-Buildings/Resources/Scripts/Dymola/Airflow/Multizone/BaseClasses/Examples/Interpolate.mos
+Buildings/Resources/ReferenceResults/Dymola/Buildings_Utilities_Math_Examples_Interpolate.txt
+Buildings/Resources/ReferenceResults/Dymola/Buildings_Utilities_Math_Functions_Examples_Interpolate.txt
```
Comment (mwetter) [inline Buildings/.copiedFiles.txt:2187]: We don't include reference results in the merge (as sometimes we may differ in the Dymola version)

File: Buildings/Resources/Scripts/Conversion/ConvertBuildings_from_10_to_11.0.0.mos [hunk 2]
```diff
@@ -52,3 +52,9 @@ convertClass("Buildings.Experimental.DHC.Plants.Reservoir.Borefield",
              
 // Conversion for https://github.com/lbl-srg/modelica-buildings/issues/3431
 convertClass("Buildings.Fluid.Geothermal.BuriedPipes","Buildings.Fluid.FixedResistances.BuriedPipes");
+
+// Conversion for https://github.com/ibpsa/modelica-ibpsa/issues/1844
+convertClass("Buildings.Airflow.Multizone.BaseClasses.interpolate",
+             "Buildings.Utilities.Math.Functions.interpolate");
+convertClass("Buildings.Airflow.Multizone.BaseClasses.Examples.interpolate",
+             "Buildings.Utilities.Math.Functions.Examples.interpolate");
```
Comment (mwetter) [inline Buildings/Resources/Scripts/Conversion/ConvertBuildings_from_10_to_11.0.0.mos:60]: Modelica is case sensitive, this should be changed to `Interpolate` for the example.

**Review by mwetter — CHANGES_REQUESTED** [review]
See inline comments.

File: Buildings/.copiedFiles.txt (hunk 1 above)
Comment (hcasperfu) [reply Buildings/.copiedFiles.txt]: Fixed in 10f14cb.
The result files in `.copiedFiles.txt` were automatically generated by the merge script.

File: Buildings/Resources/Scripts/Conversion/ConvertBuildings_from_10_to_11.0.0.mos (hunk 2 above)
Comment (hcasperfu) [reply Buildings/Resources/Scripts/Conversion/ConvertBuildings_from_10_to_11.0.0.mos]: Fixed in e1858c5.

**Pull request merged**